docs/doc_needed/run.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# IMPORTANT: the binary file of zenroom has to be in zenroom/src/
+
+# create the file needed to store the data
+>introspection.json
+>documented.txt
```

Comment:
What do you mean with create? If you do `>>` and the file doesn't exist it is created